examples/samplers/MLDA_variance_reduction_linear_regression.ipynb

```diff
@@ -533,7 +533,7 @@
     }
```

Comment:
<p>Should probably add the model context to this and below to future proof this and get rid of the annoying warning.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Sounds great. I quite didn't understand what the FutureWarning recommended, but I am happy to update the description as a PR. </p><p> </p><p>Here was my take on what the FutureWarning recommended haha:</p><p> </p><pre class="ql-syntax" spellcheck="false">with model:
&nbsp; &nbsp; trace2_az = az.from_pymc3(trace2)
az.summary(trace2_az)
</pre><p> </p>